frontend/src/routes/+layout.svelte

```diff
@@ -60,6 +60,6 @@
 </div>
 
 <!-- We don't want the alert for "hook" errors, because they land on +error.svelte -->
-{#if !$page.error?.source.endsWith('-hook')}
+{#if !$page.error?.source?.endsWith('-hook')}
```

Comment:
I'd be quite interested to know if you got an error without a `source`. I've been trying to make sure that they all get "stamped" with a source, 'cause it interests me where they're coming from.

Reply:
Not actually sure what was going on anymore, Robin just told me to do that haha